src/etc/rc.recover

```diff
@@ -62,6 +62,7 @@ audit:*:77:
 www:*:80:
 _ypldap:*:160:
 hast:*:845:
+voucher:*:1003:squid
```

Comment:
this is the freebsd template, we should not clobber it

proxy has some legacy workaround already here:

https://github.com/opnsense/core/blob/master/src/etc/inc/system.inc#L1143-L1145

But it looks like an authentication framework pre-command so not the best place in loading kernel modules